src/provisioner/provisioner.py

```diff
@@ -226,9 +220,15 @@ def _provision_components(cortx_conf: MappedConf, interfaces: Enum, apply_phase:
                         continue
                 CortxProvisioner._update_provisioning_status(
                         cortx_conf, node_id, apply_phase, ProvisionerStatus.PROGRESS.value)
-                cmd = (
-                    f"/opt/seagate/cortx/{component_name}/bin/{component_name}_setup {interface.value}"
-                    f" --config {cortx_conf._conf_url} --services {service}")
+                if interface.value == 'upgrade':
+                    # TODO: Remove config parameter for upgrade once all components will have --delta implementation
+                    cmd = (
+                        f"/opt/seagate/cortx/{component_name}/bin/{component_name}_setup {interface.value}"
```

Comment:
Pls udpate confluence spec with this change and publish to all with this patch.

Reply:
Have created confluence page : https://seagate-systems.atlassian.net/wiki/spaces/PT/pages/1011548247/Config+Upgrade , We are going to setup a call with components to convey upgrade flow with delta.